assets/stylesheets/foreground.css

```diff
@@ -884,4 +885,129 @@ body.mw-special-Userlogin h2.title {
 
 #content .oo-ui-toolbar-actions > .oo-ui-buttonElement.oo-ui-labelElement > .oo-ui-buttonElement-button > .oo-ui-labelElement-label {
     line-height: 2.75em !important;
-}
\ No newline at end of file
+}
+
+/* Add styles to Special:Preferences */
+
+/**
+ * The following code is highly modified from monobook. It would be nice if the
+ * preftoc id was more human readable like preferences-toc for instance,
+ * howerver this would require backporting the other skins.
+ */
+.client-js #preftoc {
+  /* Tabs */
+
+  width: 100%;
+  float: left;
+  clear: both;
+  margin: 0;
+  padding: 0;
+  background-position: bottom left;
+  background-repeat: no-repeat;
+}
+.client-js #preftoc li {
+  /* Tab */
+
+  float: left;
+  margin: 0;
+  padding: 0;
+  padding-right: 1px;
+  height: 2.25em;
+  white-space: nowrap;
+  list-style-type: none;
+  list-style-image: none;
+  background-position: bottom right;
+  background-repeat: no-repeat;
+  /* Sadly, IE6 won't understand this */
+
+}
+.client-js #preftoc li:first-child {
+  margin-left: 1px;
+}
+.client-js #preftoc li.selected a {
+  background-color: #fafafa;
+  border-style: solid;
+  border-width: 1px;
+  border-image: linear-gradient(to top, #cccccc, rgba(0, 0, 0, 0)) 1 100%;
+  margin: -1px;
+  background-position: bottom;
+  background-repeat: repeat-x;
+  color: #333;
+}
+.client-js #preftoc a,
+.client-js #preftoc a:active {
+  display: inline-block;
+  position: relative;
+  color: #0645ad;
+  padding: 0.5em;
+  text-decoration: none;
+  background-image: none;
+  font-size: 0.9em;
+}
+.client-js #preftoc a:hover,
+.client-js #preftoc a:focus {
+  text-decoration: underline;
+}
+.client-js #preferences {
+  float: left;
+  width: 100%;
+  margin: 0;
+  clear: both;
+  border: solid 1px #ccc;
+  background-color: #fafafa;
+}
+.client-js #preferences fieldset {
+  border: none;
+  border-top: solid 1px #ccc;
+}
+.client-js #preferences > fieldset {
+  border: none;
+  padding: 0;
+  margin: 1em;
+}
+.client-js #preferences > fieldset > legend {
+  display: none;
+}
+.client-js #preferences legend {
+  color: #666;
+}
+.client-js #preferences td {
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+.client-js #preferences div.mw-prefs-buttons {
+  padding: 1em;
+}
+.client-js #preferences div.mw-prefs-buttons input {
+  margin-right: 0.25em;
+}
+.htmlform-tip {
+  font-size: x-small;
+  padding: .2em 2em;
+  color: #666;
+}
+fieldset legend {
+  background: none !important;
+}
+
```

Comment:
preferences better :+1:  I have some code I think we can merge together for preferences improve it more on mobile too.